### PR TITLE
List more fixed bugs by mods

### LINKED ISSUE #11: Collect more bugs fixed by mods
https://github.com/Fabulously-Optimized/wiki/blob/main/en-us/changed-options.md
should list other mods that fix bugs, e.g. tooltip fix, better item edges etc, along with the Mojang bug tracker link.
PRs are welcome.
Conflicts with #9

### PR DESCRIPTION
**Pull request by LarryB123**
Fixes #11 Not finished yet.

### CODE REVIEW

**Comment by RaptaG** [comment]
Confilcts my PR :(
But it’s ok lol it gives more info

**Comment by RaptaG** [comment]
Make sure to specify the FO version of mods that aren’t used in the latest version of the modpack

**Comment by LarryB123** [comment]
I can't seem to track down anymore of mods related bugtracker entries. So this seems to be ready for review.

File: en-us/changed-options.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ However, [similar to the mod inclusion policy](principles.md), Fabulously Optimi
...
 | [MC-237493](https://bugs.mojang.com/browse/MC-237493) | Telemetry cannot be disabled                                               | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)   |
 | [MC-249059](https://bugs.mojang.com/browse/MC-249059) | Loading terrain screen cannot close before 2 seconds have passed           | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
+| [MC-73186](https://bugs.mojang.com/browse/MC-73186)   | Gaps between the faces of item models                                      |[Item Model Fix](https://www.curseforge.com/minecraft/mc-mods/item-model-fix)             |
+| [MC-26757](https://bugs.mojang.com/browse/MC-26757)   | Stops tooltips that are too large to fit on the screen from running off the screen.                                    |[ToolTipFix](https://www.curseforge.com/minecraft/mc-mods/tooltipfix)             |
+| [MC-5056](https://bugs.mojang.com/browse/MC-5056)   | Removes the potion inventory offset                                      |[No Potion Offset](https://www.curseforge.com/minecraft/mc-mods/no-potion-offset)
```
Comment (Madis0) [inline en-us/changed-options.md:63]: This can be removed, it's outdated

File: en-us/changed-options.md [hunk 2]
```diff
@@ -58,6 +58,15 @@ However, [similar to the mod inclusion policy](principles.md), Fabulously Optimi
 | [MC-235035](https://bugs.mojang.com/browse/MC-235035) | Sleeping in a custom dimension with "natural" set to false causes crash    | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
 | [MC-237493](https://bugs.mojang.com/browse/MC-237493) | Telemetry cannot be disabled                                               | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)   |
 | [MC-249059](https://bugs.mojang.com/browse/MC-249059) | Loading terrain screen cannot close before 2 seconds have passed           | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
+| [MC-73186](https://bugs.mojang.com/browse/MC-73186)   | Gaps between the faces of item models                                      |[Item Model Fix](https://www.curseforge.com/minecraft/mc-mods/item-model-fix)             |
```
Comment (Madis0) [inline en-us/changed-options.md:61]: Please order all bugs by the bug number.

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: en-us/changed-options.md (hunk 2 above)
Comment (LarryB123) [reply en-us/changed-options.md]: Resolved

**Comment by Madis0** [comment]
Thank you!

**Pull request merged**